buildenv/jenkins/getDependency

```diff
@@ -21,8 +21,8 @@ def testBuild() {
 				sh 'mkdir ${WORKSPACE}/j2sdk-image'
 				sh 'tar -xzf OpenJDK8U-jdk_x64_linux_hotspot*.gz -C ${WORKSPACE}/j2sdk-image --strip-components 1'
 				sh '${WORKSPACE}/j2sdk-image/jre/bin/java -version'
-				sh 'git clone https://github.com/AdoptOpenJDK/openjdk-systemtest'
-				sh 'git clone https://github.com/AdoptOpenJDK/stf'
+				sh 'git clone https://github.com/adoptium/aqa-systemtest'
+				sh 'git clone https://github.com/adoptium/STF'
 				sh 'ant -f ./openjdk-systemtest/openjdk.build/build.xml -Djava.home=${WORKSPACE}/j2sdk-image/jre -Dprereqs_root=${WORKSPACE}/systemtest_prereqs configure'
```

Comment:
`openjdk-systemtest` needs to be updated to `aqa-systemtest`. The same for line 27